bytestring.cabal

```diff
@@ -96,7 +96,7 @@ library
                      Data.ByteString.Builder.Prim.Internal.Floating
                      Data.ByteString.Builder.Prim.Internal.Base16
 
-  default-language:  Haskell98
+  default-language:  Haskell2010
```

Comment:
What's the motivation for this change?

Reply:
This is not critical, just to be in line with other components. 

I guesstimate the majority of developers never used `default-language: Haskell98`, so `Haskell2010` provides a more familiar environment. Besides several more extensions available by default, there are some relatively obscure differences, see https://ghc.gitlab.haskell.org/ghc/doc/users_guide/bugs.html#context-free-syntax

Reply:
Yeah, providing a familiar dev environment sounds like a good idea. 👍